bluebutton-data-model-rif/src/main/resources/db/scripts/Add_Final_Action_Column.sql

```diff
@@ -0,0 +1,33 @@
+/*
+ * This script will add the finalAction column to the below tables with a default of "F"
+ */
+
+
+-- Add finalAction column to below tables.
+
+  
+alter table "CarrierClaims" 
+   add column "finalAction" char(1) not null default "F";
```

Comment:
Just a note: we'll eventually need to get the columns ordered correctly. That's a problem with all the other columns already there, though, so oh well.